pcsx2/Gif.cpp

```diff
@@ -486,7 +486,6 @@ static u16 QWCinGIFMFIFO(u32 DrainADDR)
 		gifstate |= GIF_STATE_EMPTY;
 
 	GIF_LOG("%x Available of the %x requested", ret, gifch.qwc);
-	pxAssert(ret >= 0);
```

Comment:
I was a bit wary of removing it - I didn't see why it was added, and GCC flagged it as always true, but I didn't think it would be added without a reason. I figured it would be simpler to add it now, and rebase later if needed.